scripts/conversionScripts/convert_utils.py

```diff
@@ -110,10 +111,10 @@ def standardMain(argv,convert):
 
       tree = ET.ElementTree(ET.fromstring(strfile))
       if not always_rewrite:
-        tree_copy = ET.tostring(tree.getroot())
+        tree_copy = prettify(tree)
       convert(tree,fileName=fname)
       if not always_rewrite:
-        if ET.tostring(tree.getroot()) == tree_copy:
+        if prettify(tree) == tree_copy:
```

Comment:
the preffify is used here because it standardizes the appearance of the XML file